cli/tests/test_cli_smoke.py

```
@@ -53,6 +56,343 @@ def test_init_help_works(self):
+            for name in ("SystemRoot", "WINDIR", "COMSPEC", "PATHEXT", "TEMP", "TMP"):
+                if value := os.environ.get(name):
+                    environment[name] = value
+
+            def invoke(
+                *arguments: str,
+                gateway_required: bool = False,
+            ) -> subprocess.CompletedProcess[str]:
+                child_environment = environment.copy()
+                if not gateway_required:
+                    child_environment.pop("DEFENSECLAW_GATEWAY_BIN", None)
+                return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)